app/package.json

```diff
@@ -11,8 +11,16 @@
     "cypress:e2e": "cypress run --e2e"
   },
   "dependencies": {
+    "@bcgov/bc-sans": "^1.0.1",
+    "@emotion/react": "^11.11.0",
```

Comment:
You should remove the caret symobols `^` from your package.json dependencies and run the command `npm config set save-exact true` so this doesnt happen again.

Reply:
Will do.